tests/common.go

```diff
@@ -703,6 +707,38 @@ func ValidateContextForPureVolumesSDK(ctx *scheduler.Context, errChan ...*chan e
 				})
 			}
 		})
+
+		//check mount option attribute in storage class
+		Step("validating mount options", func() {
+			if ctx.SkipMountOptionValidation {
+				return
+			}
+			ctx.SkipVolumeValidation = false
+			ValidateContext(ctx)
+			ctx.SkipVolumeValidation = true
+			vols, err := Inst().S.GetVolumes(ctx)
+			processError(err, errChan...)
+			for _, vol := range vols {
+				pvcObj, err := k8sCore.GetPersistentVolumeClaim(vol.Name, vol.Namespace)
+				if err != nil {
+					log.FailOnError(err, " Failed to get pvc for volume %s", vol)
+				}
+				sc, err := k8sCore.GetStorageClassForPVC(pvcObj)
+				if err != nil {
+					log.FailOnError(err, " Error Occured while getting storage class for pvc %s", pvcObj)
+				}
+				sc2 := fmt.Sprint(sc)
```

Comment:
[nit] You can inline the `fmt.Sprint(sc)` in the below command, the `sc2` variable feels extra for being a type conversion being used in one place

Reply:
Done, now we using MountOption attribute from sc to check nosuid flag.